The windows runners fail when re-installing MDAnalysis. It looks as if the mamba line just doesn't do anything:
```
Install line: MDAnalysis MDAnalysisTests
Run mamba install -y -c conda-forge MDAnalysis MDAnalysisTests
  mamba install -y -c conda-forge MDAnalysis MDAnalysisTests
  shell: C:\Program Files\Git\bin\bash.EXE -l {0}
  env:
    INPUT_RUN_POST: true
    CONDA: C:\Users\runneradmin\miniconda3
Microsoft Windows [Version 10.0.26100.32522]
(c) Microsoft Corporation. All rights reserved.

D:\a\mdaencore\mdaencore>
```
The above _should_ install something, right?

When we then try to import
```
Run pyver=$(python -c 'import MDAnalysis; print(MDAnalysis.__version__)')
Traceback (most recent call last):
  File "<string>", line 1, in <module>
    import MDAnalysis; print(MDAnalysis.__version__)
    ^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'MDAnalysis'
Error: Process completed with exit code 1.
```
then there's no MDAnalysis package.

--- a/.github/workflows/gh-ci.yaml
+++ b/.github/workflows/gh-ci.yaml
@@ -72,7 +72,7 @@ jobs:
       with:
         version: ${{ matrix.mdanalysis-version }}
         install-tests: true
-        installer:  mamba 
+        installer:  conda 
         shell: bash  -l {0} 
       
     - name: Install package

--- a/.github/workflows/gh-ci.yaml
+++ b/.github/workflows/gh-ci.yaml
@@ -73,7 +73,6 @@ jobs:
         version: ${{ matrix.mdanalysis-version }}
         install-tests: true
         installer:  mamba 
-        shell: bash  -l {0} 
       
     - name: Install package
       run: |

--- a/.github/workflows/gh-ci.yaml
+++ b/.github/workflows/gh-ci.yaml
@@ -73,6 +73,7 @@ jobs:
         version: ${{ matrix.mdanalysis-version }}
         install-tests: true
         installer:  mamba 
+        shell: bash  -l {0} 
       
     - name: Install package
       run: |

